pkg/controller/master/backup/backup_metadata.go

```diff
@@ -152,6 +153,12 @@ func (h *MetadataHandler) OnBackupTargetChange(_ string, setting *harvesterv1.Se
 		return nil, nil
 	}
 
+	contextLogger.Info("check existing vm backup...")
+	if err = h.checkExistingVMBackup(target); err != nil {
+		contextLogger.WithError(err).Errorf("can't check existing vm backup")
```

Comment:
It may have possible performance penalty:

When refreshIntervalInSeconds is set, in each refresh interval, all backups are checked, not in an incremental mode

Reply:
We cannot do it in an incremental mode. If a VMBackup is heathy in this check and a data is removed unexpectedly after 5 seconds, how do we know this VMBackup is unhealthy? If using incremental mode, we skip this VMBackup check. So we finally need `refreshIntervalInSeconds` to check all VMBackup again.